e2e/utils.go

```diff
@@ -585,6 +585,23 @@ func validatePVCClone(totalCount int, sourcePvcPath, sourceAppPath, clonePvcPath
 				LabelSelector: fmt.Sprintf("%s=%s", appKey, label[appKey]),
 			}
 			wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+			if wgErrs[n] == nil && kms != "" {
+				if kmsIsVault(kms) || kms == vaultTokens {
+					imageData, sErr := getImageInfoFromPVC(p.Namespace, name, f)
+					if sErr != nil {
+						wgErrs[n] = fmt.Errorf("failed to get image info for %s namespace=%s volumehandle=%s error=%w", name, p.Namespace, imageData.csiVolumeHandle, sErr)
+					} else {
+						// check new passphrase created
+						stdOut, stdErr := readVaultSecret(imageData.csiVolumeHandle, kmsIsVault(kms), f)
```

Comment:
@yati1998 I would check for `stdErr` first , thus we can  avoid  check of `stdOut` ?

Reply:
Hey, was just going through the code again, but I don't think it matters which check we keep first. Because even if there is any failure it still checks for stdOut, as the error is logged in wgErrs[n] and doesn't fail.